packages/admin/admin/src/mui/menu/ItemAnchorLink.tsx

```diff
@@ -6,23 +6,20 @@ import { MenuItem, MenuItemProps } from "./Item";
 
 export type MenuItemAnchorLinkProps = MenuItemProps & ListItemButtonProps & HTMLProps<HTMLAnchorElement>;
 
-export const MenuItemAnchorLink: React.FC<MenuItemAnchorLinkProps> = ({ secondaryAction, isMenuOpen, slotProps, ...props }) => {
-    const computedSecondaryAction =
-        secondaryAction !== undefined ? (
-            secondaryAction
+export const MenuItemAnchorLink = ({ secondaryAction: passedSecondaryAction, isMenuOpen, slotProps, ...props }: MenuItemAnchorLinkProps) => {
+    const secondaryAction =
+        passedSecondaryAction !== undefined ? ( // don't use ?? to allow null as value and with that an empty secondaryAction
```

Comment:
The nullish coalescing operator allows null or undefined: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
Yeah and I don't want that because if the passed secondaryAction is undefined it should go to the fallback default but if null is passed i would assume that the user doesn't want the secondaryAction to show up at all

Reply:
Ah, misread that. Sorry